### PR TITLE
SMTP against ImprovMX

### PR DESCRIPTION
**Pull request by Matthijsy**
Let's create the SMTP accounts in ImprovMX as well

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -62,7 +62,7 @@ class Application < Rails::Application
     config.x.mailgun_api_key = credentials.dig(Rails.env.to_sym, :mailgun_api_key)
     config.x.mailgun_validation_key = credentials.dig(Rails.env.to_sym, :mailgun_validation_key)
     config.x.mailgun_host = 'api.eu.mailgun.net'
-    config.x.mail_domains = %w[csvalpha.nl societeitflux.nl sandbox86621.eu.mailgun.org]
+    config.x.mail_domains = %w[csvalpha.nl societeitflux.nl sandbox86621.eu.mailgun.org hetkrat.nl]
```
Comment (wilco375) [inline config/application.rb:65]: Why this change?

**Review by wilco375 — CHANGES_REQUESTED** [review]

**Comment by wilco375** [comment]
One small question, looks good otherwise

File: config/application.rb (hunk 1 above)
Comment (Matthijsy) [reply config/application.rb]: Whoops that was for testing only

**Review by wilco375 — APPROVED** [review]
